101-streamanalytics-create/azuredeploy.json

```diff
@@ -0,0 +1,65 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.1.0.0",
+  "parameters": {
+    "streamAnalyticsJobName": {
+      "type": "string",
+      "minLength": 3,
+      "maxLength": 63,
+      "metadata": {
+        "description": "Stream Analytics Job Name, can contain alphanumeric characters and hypen and must be 3-63 characters long"
+      }
+    },
+    "numberOfStreamingUnits": {
+      "type": "int",
+      "minValue": 1,
+      "maxValue": 48,
+      "allowedValues": [
+        1,
+        3,
+        6,
+        12,
+        18,
+        24,
+        30,
+        36,
+        42,
+        48
+      ],
+      "defaultValue": 1,
+      "metadata": {
+        "description": "Number of Streaming Units"
+      }
+    }
+  },
+  "variables": {
```

Comment:
since these are only used once just use literal values in the resource properties